package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@optbot/pytools",
   "version": "0.0.0",
```

Comment:
If the service doesn't exist, we should imo start with 0.0.0 and definitely not with 0.1.0. In cases where there is pre-existing code, I'm not worried about the difference between 0.0.0 and 0.0.1. Months from now, if we end up splitting a service that is already at a somewhat higher number, I think we'll have to decide case by case what we want to do on version numbers, which should always be included on Trello as well to track what iteration we're on.